loader/src/main/java/net/neoforged/fml/loading/moddiscovery/ModFileParser.java

```diff
@@ -85,17 +87,28 @@ protected static List<CoreModFile> getCoreMods(final ModFile modFile) {
                 .toList();
     }
 
-    protected static List<String> getMixinConfigs(IModFileInfo modFileInfo) {
+    /**
+     * Represents a potential mixin configuration.
+     *
+     * @param name The name of the mixin configuration.
+     * @param requiredModIds The mod ids that are required for this mixin configuration to be loaded. If empty, will be loaded regardless.
+     */
+    public record PotentialMixinConfig(String name, List<String> requiredModIds) {}
```

Comment:
Since this is API, I would probably call this differently. It really is just a Java object model for how a mod declares a mixin config. Maybe `MixinsEntry` or whatever.

Reply:
Given that it is not API (else JCC would not allow this), the naming does not matter, but MixinsEntry is fine to me.

Reply:
Actually I named it `MixinConfig` as the method is named: `getMixinConfigs`